### PR TITLE
Fix build script

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by boegel** [comment]
@stdweird Please add some info on the use of `$INSTALLDEPS` and `$WEBSERVER` to control `build.sh` to the README?

Or maybe it's better to use proper CLI options for the script, so you can do `./build.sh --install-deps --preview` or whatever?

File: build.sh [hunk 1]
```diff
@@ -1,5 +1,28 @@
 #!/bin/bash
-# python3 -m pip install --ignore-installed --prefix $PWD/pypkgs -r requirements.txt
-# python3 -m pip install --prefix $PWD/pypkgs $PWD/custom_plugin
-# python3 -m pip install --prefix $PWD/pypkgs $PWD/computational_macros
-PYTHONPATH=$PWD/pypkgs/lib/python3.9/site-packages:$PWD/custom_plugin:$PWD/computational_macros:$PYTHONPATH python3 build.py
+set -x
+if [ ${INSTALLDEPS:-0} -gt 0 ]; then
+    python3 -m pip install --ignore-installed --prefix $PWD/pypkgs -r requirements.txt
```
Comment (alvarosimon) [inline build.sh:4]: `shellcheck build.sh`  shows several warnings such using  double quotes for all variables like "$PWD"

**Comment by stdweird** [comment]
@boegel don't know what is best. if we add proper cli options, we need proper --help; and now i just pass the cli opts to the build.py
i'll add them to the README for now

**Comment by stdweird** [comment]
@boegel i'm assuming that the mkdocs commands are independent

**Pull request closed without merging**